.gitlab-ci.yml

```diff
@@ -347,6 +348,35 @@ doc:refman:dune:
     paths:
       - _build/default/doc/sphinx_build/html
 
+doc:refman:deploy:
+  stage: deploy
+  environment:
+    name: deployment
+    url: https://coq.github.io/
+  only:
+    variables:
+      - $DOCUMENTATION_DEPLOY_KEY
+  dependencies:
+    - doc:refman:dune
+  before_script:
+    - which ssh-agent || ( apt-get update -y && apt-get install openssh-client -y )
+    - eval $(ssh-agent -s)
+    - echo "$DOCUMENTATION_DEPLOY_KEY" | tr -d '\r' | ssh-add - > /dev/null
+    - mkdir -p ~/.ssh
+    - chmod 700 ~/.ssh
+    - ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+    - git config --global user.name "coqbot"
+    - git config --global user.email "coqbot@users.noreply.github.com"
+  script:
+    - git clone git@github.com:coq/doc.git
+    - rm -rf doc/$CI_COMMIT_REF_NAME/refman # NB: won’t work if branch name has ‘/’ in it
+    - mkdir -p doc/$CI_COMMIT_REF_NAME
```

Comment:
This command does not look future proof.

Reply:
>This command does not look future proof.

How so?

Reply:
Sorry for the noise. I misread that in `rmdir`. I'm too tired.